### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_Inspect: Name is incorrect for Upload button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3970]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106
Operating System:Windows10

**Repro Steps:**
1.Login with the appropriate credentials to azure notebook account.
2.Navigate to run button & hit enter --> Navigate to Azure Notebooks welcome & hit enter.
3.Verify name of "Upload" button.

**Actual:**
Name is incorrect for the "Upload" button.

**Expected:**
Name for the  "Upload" button should be "Upload".

**User Impact:**
Screen reader user would get confused when screen reader reading name of button is incorrect.

Attachment:
![a11y_azurenotebooks_jupyter_filestab_inspect jpg](https://user-images.githubusercontent.com/39204909/45801913-f8e96700-bcd2-11e8-92ab-040fccfca0cc.png)

**Comment by takluyver** [comment]
Closed by #4689
